Sources/PackageGraph/DependencyResolver.swift

```diff
@@ -309,7 +365,7 @@ struct PackageContainerConstraintSet<C: PackageContainer>: Collection {
     {
         var result = self
         for (key, versionRequirement) in constraints {
-            guard let merged = result.merging(versionRequirement: versionRequirement, for: key) else {
+            guard let merged = merging(requirement: versionRequirement, for: key) else {
```

Comment:
This is a bug, I think? `result.merging` -> `merging`? We aren't accumulating the results for all constraints... unfortunate no test showed that.